src/analyze/analyze.c

```diff
@@ -535,6 +539,10 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_legend = false;
                         break;
 
+                case ARG_TLDR:
+                        arg_cat_flags = CAT_FLAGS_SUPPRESS_COMMENTS;
```

Comment:
so, this is very specific to our regular conf file format (and not really to tmpfiles.d/ files), hence I think this flag should be named after that fact.

or maybe make this two separate flags: CAT_SUPPRESS_HASH_COMMENTS and CAT_SUPPRESS_REDUNDANT_BRACKET_SECTIONS, so that we set both when we look at conf files, and only one when we look a tmpfiles.d/sysusers.d and suchlike.

Reply:
and CAT_SUPPRESS_SEMICOLON_COMMENTS